src/components/molecules/button/button.types.ts

```diff
@@ -0,0 +1,2 @@
+export const sizesAllowed = ['small', 'medium', 'large']
+export const variantAllowed = ['contained', 'outlined', 'text']
```

Comment:
use these arrays to have initial values `size = sizesAllowed[0]`

Reply:
done!